### ISSUE TITLE
About iterative_test, I have test the traning procedure with iteration%100=0 and after 60000 start  iteration%100=0 validation. Strange tihing that without iterative val the loss is different, whereas if  with iterative val the loss is same.

### ISSUE THREAD

**Issue by GingerCohle** [CityU-AIM-Group/SIGMA#53]
Then I notice that
![image](https://github.com/CityU-AIM-Group/SIGMA/assets/37873318/b919149b-e52a-43f6-94ee-ef72b4d7010c)
.Could you explain why?

**Comment by wymanCV** [comment]
Hi,  we control the random seed, but I am not sure whether some operations may lead to slight difference with the same random seed, such as different times of model.eval(), batch sampler, and dataloader.  Sometimes, the loss is also different when I run on different machines. It shouldn't make much difference, I think.

**Comment by wymanCV** [comment]
I just checked that all parameters, including BN, are frozen and grad is not computed in val stage. If you find the solutions, I would be happy if you could share them with us.

**Comment by GingerCohle** [comment]
ok. thank you

**Comment by wymanCV** [comment]
@GingerCohle  Thanks for your attention to our work. You can add me via WeChat 17720031102.
